integration_tests/mockApis/responses/imprisonmentStatuses.ts

```diff
@@ -0,0 +1,225 @@
+export default [
+  {
+    code: 'on-remand',
```

Comment:
code is unique to each status. It is also the same as the last part of the url. We can therefore use the code for navigation purposes